core/db/migrate/20160511072249_change_collation_for_tag_names.rb

```diff
@@ -0,0 +1,9 @@
+# This migration is added to circumvent issue #623 and have special characters
+# work properly
+class ChangeCollationForTagNames < ActiveRecord::Migration
+  def up
+    if ActsAsTaggableOn::Utils.using_mysql?
+      execute("ALTER TABLE spree_tags MODIFY name varchar(255) CHARACTER SET utf8 COLLATE utf8_bin;")
```

Comment:
@krzysiek1507  If you checked log from travis-ci its executing ```execute("ALTER TABLE tags MODIFY name varchar(255) CHARACTER SET utf8 COLLATE utf8_bin;")``` command which is not present in this migration, Yes but this is in gem ```acts_as_taggable```.